packages/webdriverio/tests/commands/element/selectByVisibleText.test.js

```diff
@@ -0,0 +1,33 @@
+import request from 'request'
+import { remote } from '../../../src'
+
+describe('selectByVisibleText test', () => {
+    let browser
+    let elem
+
+    beforeAll(async () => {
+        browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+        elem = await browser.$('some-elem-123')
+    })
+
+    it('should select value by visible text', async () => {
+        await elem.selectByVisibleText('someValue1')
+
+        expect(request.mock.calls[1][0].uri.path).toBe('/wd/hub/session/foobar-123/element')
```

Comment:
Maybe worth adding a check that parameter were normalized correctly, similar for the other commands